lazy_tensor_core/lazy_tensor_core/csrc/init_python_bindings.cpp

```diff
@@ -54,7 +54,7 @@ c10::optional<Device> GetOptionalDevice(const std::string& device_str) {
 
 Device GetDeviceOrCurrent(const std::string& device_str) {
   if (device_str.empty()) {
-    return GetCurrentDevice();
+    return Device();
```

Comment:
so what does this do exactly?

Reply:
The default `Device` constructor should set the default device type according to the backend interface and set the ordinal to zero (which can be changed if later on we add back API to support default device).

Reply:
> Device constructor should set the default device type according to the backend interface 
is this something you planned to add later?  Or i missed it and it's already there?

Reply:
Added a TODO in the default ctor' impl. Holding that on your backend API change.